### PR TITLE
Add postgresql support

### PR DESCRIPTION
**Pull request by andypost**
It may need to change CI to setup `COMPOSER_HOME_CACHE` for common runner cache

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,9 +24,13 @@ CGID ?= $(LOCAL_GID)
 # Define network name.
 COMPOSE_NET_NAME := $(COMPOSE_PROJECT_NAME)_front
 
-# Determine mysql data directory if defined
+SDC_SERVICES=$(shell docker-compose config --services)
```
Comment (andypost) [inline Makefile:27]: needs to use it

**Comment by davidferlay** [comment]
Issue to fix 
image

File: Makefile [hunk 2]
```diff
@@ -148,9 +154,13 @@ clean: info
...
 	$(eval SCAFFOLD = $(shell docker run --rm -v $(CURDIR):/mnt -w /mnt --user $(CUID):$(CGID) $(IMAGE_PHP) composer run-script list-scaffold-files | grep -P '^(?!>)'))
 	@docker run --rm --user 0:0 -v $(CURDIR):/mnt -w /mnt -e RMLIST="$(addprefix web/,$(SCAFFOLD)) $(DIRS)" $(IMAGE_PHP) sh -c 'for i in $$RMLIST; do rm -fr $$i && echo "Removed $$i"; done'
-ifdef MYSQL_DIR
-	@echo "Removing mysql data from $(MYSQL_DIR) ..."
-	docker run --rm --user 0:0 -v $(shell dirname $(MYSQL_DIR)):/mnt $(IMAGE_PHP) sh -c "rm -fr /mnt/`basename $(MYSQL_DIR)`"
+ifdef DB_MOUNT_DIR
+	@echo "Clean-up database data from $(DB_MOUNT_DIR) ..."
+	docker run --rm --user 0:0 -v $(shell dirname $(DB_MOUNT_DIR)):/mnt $(IMAGE_PHP) sh -c "rm -fr /mnt/`basename $(DB_MOUNT_DIR)`"
+endif
+ifdef COMPOSER_HOME_CACHE
```
Comment (andypost) [inline Makefile:161]: Cache should not be cleaned as it used in CI as shared

File: .env.default [hunk 3]
```diff
@@ -18,11 +18,14 @@ IMAGE_REDIS=redis:5-alpine
 IMAGE_DRIVER=zenika/alpine-chrome
 CLEAR_FRONT_PACKAGES=no
 ADD_PHP_EXT=
+#ADD_PHP_EXT=php7-pdo_pgsql postgresql-client postgresql-contrib  gnu-libiconv wkhtmltopdf
+COMPOSER_HOME_CACHE=.cache/composer
```
Comment (andypost) [inline .env.default:22]: Needs to check if path relative or full

Comment (andypost) [reply .env.default]: Composer changed removed in favour of https://github.com/skilld-labs/skilld-docker-container/pull/278

**Pull request merged**